setup.py

```diff
@@ -31,14 +31,13 @@
         'scripts/pre-commit.git-lint.sh',
         'scripts/pre-commit.hg-lint.sh',
         'scripts/custom_linters/ini_linter.py',
-        'scripts/custom_linters/yaml_linter.py',
         'scripts/custom_linters/jpegtran-linter.sh',
         'scripts/custom_linters/optipng-linter.sh',
         'scripts/custom_linters/pngcrush-linter.sh',
         'scripts/custom_linters/tidy-wrapper.sh',
     ],
     install_requires=[
-        'pyyaml',
+        'yamllint',
```

Comment:
Please move this to the section below (after the comment), where I have all the linters that are pip installable.